.github/workflows/merge_upstream_master.yml

```
@@ -93,11 +86,18 @@ jobs:
 
           git fetch origin "$PR_BRANCH" --depth=$((ahead_by + 1))
           git fetch upstream "$BASE_BRANCH" --depth=$((behind_by + 1))
+
+          # Check if a workflow file would be modified by the merge (permissions prevent pushes if so)
+          latest_workflow_commit=$(git log -n 1 --pretty=format:"%H" upstream/$BASE_BRANCH -- .github/workflows)
+          if ! git branch --contains $latest_workflow_commit | grep -q "$(git rev-parse --abbrev-ref HEAD)"; then
+            gh pr comment ${{ github.event.issue.html_url }} --body "GitHub Actions can not push to this branch as workflow files have been changed since your branch was last updated. Please update your branch past https://github.com/ParadiseSS13/Paradise/commit/$latest_workflow_commit before using this command again."
+            exit 0
+          fi
+
           git merge FETCH_HEAD
           git push origin
 
       - name: Notify Failure
         if: failure()
-        env:
-          GH_TOKEN: ${{ steps.create_token.outputs.token }}
-        run: gh pr comment ${{ github.event.issue.html_url }} -b "Merging upstream failed:\nhttps://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
+        run: |
+          gh pr comment ${{ github.event.issue.html_url }} -b "Merging upstream failed, see the action run log for details: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```

```suggestion
        run: gh pr comment ${{ github.event.issue.html_url }} -b "Merging upstream failed, see the action run log for details: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
```
Multiline remnant of a removed system, yeet.